MessageBird/Objects/Conversations/ConversationStartRequest.cs

```diff
@@ -0,0 +1,20 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace MessageBird.Objects.Conversations
+{
+    public class ConversationStartRequest
+    {
+        [JsonProperty("to")]
+        public string To { get; set; }
+        
+        [JsonProperty("type"), JsonConverter(typeof(StringEnumConverter))]
+        public ContentType Type {get; set;}
```

Comment:
`ContentType`?

Reply:
I think it's good to follow the MB API naming here (java lib does the same)